Corrected an essentially wrong use of "cluster" to "node".

wkasdorp opened the pull request:
This was my reference: https://docs.microsoft.com/en-us/azure/virtual-machines/windows/allocation-failure, thx to Rodrigo Gomes.

azurecla commented:
Hi __@dem108__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

wkasdorp committed:
Corrected an essentially wrong use of "cluster" to "node".

This was my reference: https://docs.microsoft.com/en-us/azure/virtual-machines/windows/allocation-failure, thx to Rodrigo Gomes.
